### PR TITLE
Survey sentiment and database API integration

### PR DESCRIPTION
**Pull request by yutongliuytl**
# Details
- Added Sentiment POST request after form submission
- Aggregate all info and added Database POST request
- Verified data is correct w/ Benny

### CODE REVIEW

File: frontend/src/pages/DailySurveys/index.tsx [hunk 1]
```diff
@@ -25,16 +28,85 @@ const validateMessages = {
...
+    const sentiment = await getSentiment([values.journal]);
+
+    // Constructing Database API body
+    const date = moment().format("YYYY-MM-DD");
+    const name = "Kevin";
+    const dataInput = {
+      "Name": name,
+      "Timestamp": date,
+      "Message": values.journal,
+      "Sentiment": (sentiment == "positive"? 1 : 0),
```
Comment (jx3yang) [inline frontend/src/pages/DailySurveys/index.tsx]: Sentiment can be positive, neutral, or negative from the model. Do we want to make it binary?

Comment (yutongliuytl) [reply frontend/src/pages/DailySurveys/index.tsx]: Didn't know that, will notice Benny as well

Comment (jbennyhuang) [reply frontend/src/pages/DailySurveys/index.tsx]: Ill make it into string from the db side

**Review by jbennyhuang — APPROVED** [review]
I changed my field to string. It should be fine now.

**Pull request merged**